app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    @comment.save
+
+    redirect_to company_job_path(@comment.job.company_id, @comment.job_id)
```

Comment:
can reference `company_id` and `job_id` by the object as well. so just `company` and `job`